src/support/lockedpool.cpp

```diff
@@ -27,6 +27,10 @@
 #endif
 
 #include <algorithm>
+#ifdef ARENA_DEBUG
```

Comment:
We need `-DARENA_DEBUG` flag here https://github.com/dashpay/dash/blob/develop/ci/matrix.sh#L66 to mimic https://github.com/bitcoin/bitcoin/pull/16161/files#diff-f337b918500d409d1f297c298dcc61ed8fb6843640d76fdd42dcf3e2e5c6581dR12 from the original PR